.travis.yml

```diff
@@ -1,20 +1,24 @@
+os:
+  - linux
+  - osx
+
 language: c++
 
 compiler:
     - gcc
 
 before_install:
-  - sudo apt-get update
-  - sudo apt-get install -y git
-  - sudo apt-get install -y libboost-python-dev
-  - sudo apt-get install -y libboost-system-dev
-
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then sudo apt-get update; fi
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then sudo apt-get install -y git; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install boost-python; else sudo apt-get install -y libboost-python-dev; fi
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then sudo apt-get install -y libboost-system-dev; fi
 before_script:
   - mkdir build
   - cd build
-  - cmake ../src
-
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cmake ../src -DPYTHON_LIBRARY=$(python-config --prefix)/lib/libpython2.7.dylib -DPYTHON_INCLUDE_DIR=$(python-config --prefix)/include/python2.7 -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev; else cmake ../src; fi
```

Comment:
Note that I had to add the following flags
- `-DPYTHON_LIBRARY=$(python-config --prefix)/lib/libpython2.7.dylib`
- `-DPYTHON_INCLUDE_DIR=$(python-config --prefix)/include/python2.7`
To make FindPython find the right libs (the ones from the homebrew-installed python)

The other flags are general flags that homebrew normally adds that I got from somewhere else, not sure if necessary (probably not)